dod-advana/gamechanger-web
# Task/uot 109905 "About GC"/FAQ

## Description

-"About GC" page, this is a static page so most of it is hard coded.
-FAQ page, this pulls question, answer, and category from postgres table.

## Related Issue/Ticket

[UOT-109712](https://jira.di2e.net/browse/UOT-109712)
[UOT-109905](https://jira.di2e.net/browse/UOT-109905)

## Testing instructions

-Click on the About button in the sidebar.
-Check that everything renders correctly
-Click on FAQ tab at the top, main thing to check here is that when scrolling the sidenav changes with which category is on top. Clicking on the sidenav also should scroll to category. 
-Links in the questions should be hyperlinks
